### PR TITLE
Rclpy minimal services

### PR DESCRIPTION
**Pull request by mikaelarguedas**
Simple service examples

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
This PR adds service examples with a similar behavior as the cpp examples added in #138

File: rclpy/services/minimal_client/test/test_copyright.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ament_copyright.main import main
+
+
+def test_copyright():
+    rc = main(argv=['.', 'test'])
```
Comment (dirk-thomas) [inline rclpy/services/minimal_client/test/test_copyright.py]: Why does this need to be `.` and `test`? Isn't `test` a subfolder of the package root `.`?

Comment (mikaelarguedas) [inline rclpy/services/minimal_client/test/test_copyright.py]: updated in https://github.com/ros2/examples/pull/140/commits/4eddd82947fefc0820b26c193df932c440a08c4e see https://github.com/ros2/examples/pull/139#discussion_r90070531

File: rclpy/services/minimal_client/client.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+def main(args=None):
+    rclpy.init(args)
+    node = rclpy.create_node('minimal_client')
+    cli = node.create_client(AddTwoInts, 'add_two_ints')
+
+    req = AddTwoInts.Request()
+    req.a = 41
+    req.b = 1
+    # wait for connection to be established
+    # (no wait for service in Python yet)
```
Comment (wjwwood) [inline rclpy/services/minimal_client/client.py]: I would recommend a TODO here then, so we remember to replace this with wait for service later. Or if there is a ticket to add wait for service in Python, you could link to this line of this file (at a specific commit) as a place to update after it is added. You might want to do both.

File: rclpy/services/minimal_client/client_async.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    node = rclpy.create_node('minimal_client_async')
+
+    cli = node.create_client(AddTwoInts, 'add_two_ints')
+
+    req = AddTwoInts.Request()
+    req.a = 41
+    req.b = 1
+    # wait for connection to be established
+    # (no wait for service in Python yet)
```
Comment (wjwwood) [inline rclpy/services/minimal_client/client_async.py]: TODO here as well

File: rclpy/services/minimal_client/client_async_member_function.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import rclpy
+
+from example_interfaces.srv import AddTwoInts
+
+
+class MinimalClientAsync:
+    def __init__(self, node):
+        self.cli = node.create_client(AddTwoInts, 'add_two_ints')
+        # wait for connection to be established
+        # (no wait for service in Python yet)
```
Comment (wjwwood) [inline rclpy/services/minimal_client/client_async_member_function.py]: TODO

File: rclpy/services/minimal_service/service.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# limitations under the License.
+
+import rclpy
+
+from example_interfaces.srv import AddTwoInts
+
+
+def add_two_ints_callback(request, response):
+    response.sum = request.a + request.b
+    print('Incoming request\na: %d b:%d' % (request.a, request.b))
```
Comment (wjwwood) [inline rclpy/services/minimal_service/service.py]: nitpick: `b: %d`

File: rclpy/services/minimal_client/client.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    time.sleep(1)
+
+    cli.call(req)
+    # when calling wait for future
+    # spin should not be called in the main loop
+    cli.wait_for_future()
+    print('Result of add_two_ints: for %d + %d = %d' % (
```
Comment (dirk-thomas) [inline rclpy/services/minimal_client/client.py]: Nitpick: wrap directly after parenthesis, e.g.:

```
print(
    'Result of add_two_ints: for %d + %d = %d' %
    (req.a, req.b, cli.response.sum))
```

Same below.

File: rclpy/services/minimal_client/client_async.py [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    req = AddTwoInts.Request()
+    req.a = 41
+    req.b = 1
+    # TODO(mikaelarguedas) remove this once wait for service implemented
+    # wait for connection to be established
+    # (no wait for service in Python yet)
+    time.sleep(1)
+    cli.call(req)
+    while rclpy.ok():
+        if cli.response is not None:
```
Comment (mikaelarguedas) [inline rclpy/services/minimal_client/client_async.py]: What do you think about this TODO message? https://github.com/ros2/examples/pull/140/commits/53060743ee85c5d602ff942fada4193284573e10

Comment (dirk-thomas) [inline rclpy/services/minimal_client/client_async.py]: :+1:

**Pull request merged**